src/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -77,9 +77,12 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation):
             $instructionPriority = $this->createElement('InstrPrty', $paymentInformation->getInstructionPriority());
             $paymentTypeInformation->appendChild($instructionPriority);
         }
-        $serviceLevel = $this->createElement('SvcLvl');
-        $serviceLevel->appendChild($this->createElement('Cd', 'SEPA'));
-        $paymentTypeInformation->appendChild($serviceLevel);
+
+        if ($this->messageFormat->getMessageName() !== '001.001.03') {
```

Comment:
Please fully remove this code for the time being, not just against the 03 version

Reply:
I'm trying out this change locally! should this be `pain.001.001.03` ?
```php
if ($this->messageFormat->getMessageName() !== 'pain.001.001.03') {
```

Reply:
Yes, stupid, fixed, included a test for it.